src/Honeybee_Generator_PV.py

```diff
@@ -57,7 +57,7 @@
 
 ghenv.Component.Name = "Honeybee_Generator_PV"
 ghenv.Component.NickName = 'PVgen'
-ghenv.Component.Message = 'VER 0.0.58\nNOV_25_2015'
+ghenv.Component.Message = 'VER 0.0.58\nJAN_16_2015'
```

Comment:
Here is why the ghuser is missing. The date is wrong and Ladybug doesn't overwrite the current component which it thinks is a newer version! It should be nJAN_16_201**6**